src/main/bash/sdkman-completion.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+_sdk() {
```

Comment:
How do you feel about calling this top level function something like `_sdk_completion` or something similar? This will keep it distinct from the rest of the sdkman internal functions.

Reply:
Good idea!